src/bp/ui-studio/src/web/components/Layout/BottomPanel/Debugger/index.tsx

```diff
@@ -68,6 +71,12 @@ export class Debugger extends React.Component<Props, State> {
       await this.loadEvent(this.props.eventId)
     }
 
+    window.addEventListener('message', ({ data: { name, payload } }) => {
```

Comment:
I find it a bit weird that configs are managed here, in Settings and in ConfigEditor why can't we keep the config related code in one location ?

Reply:
Good point, I wanted to keep the config on the top most component so it's always available, but since it's a tab, the Settings component can hold it all. Will change this